docs/site/DEVELOPING.md

```diff
@@ -938,3 +939,15 @@ in progress when new commits were added. If GitHub complains that RenovateBot's
 pull request is out of date, then just wait until it's rebased and checks are
 green. The bot usually updates pull requests every hour. Alternatively, tick the
 check-box in the pull request description or in "Update Dependencies" issue.
+
+## Troubleshooting
+
+If you are using Visual Studio Code, below are some tips to help you debug the
```

Comment:
Have you considered moving this VSCode-specific content to https://github.com/strongloop/loopback-next/blob/master/docs/site/Developing-with-vscode.md?

Reply:
I didn't realize there is already a vscode developing guide. Mine is a duplicate. removed.